### PR TITLE
Added test cases for renderBooks()

### PR DESCRIPTION
**Pull request by thuyanduong-flatiron**
Fix for issue [#7](https://github.com/learn-co-curriculum/fewpjs-fetch-lab/issues/7)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 function fetchBooks() {
+  return fetch('https://anapioficeandfire.com/api/books')
+  .then(resp => resp.json())
+  .then(renderBooks)
```
Comment (sgharms) [inline index.js]: @thuyanduong-flatiron Is this the solution code? Should this be removed?

File: test/indexTest.js [hunk 2]
```diff
@@ -13,13 +12,20 @@ describe( "index.js", () => {
...
     } );
 
     it( "sends a fetch request to 'https://anapioficeandfire.com/api/books'", async () => {
+      chai.spy.on( window, 'fetch' );
       await fetchBooks()
       expect( window.fetch, "A fetch to the API was not found" )
         .to.have.been.called.with( 'https://anapioficeandfire.com/api/books' );
     } )
+
+    it( "sends renders books onto the DOM'", async () => {
```
Comment (sgharms) [inline test/indexTest.js]: Maybe change to "...into the DOM by passing a JSON object to `renderBooks`"? Then the failure is more instructive?

**Review by sgharms — CHANGES_REQUESTED** [review]
@thuyanduong-flatiron This seems good, but I wanted to check a few points before we merge.

File: index.js (hunk 1 above)
Comment (thuyanduong-flatiron) [reply index.js]: I complete didn't notice. I have removed it now.

File: test/indexTest.js (hunk 2 above)
Comment (thuyanduong-flatiron) [reply test/indexTest.js]: I have updated the testing language

**Comment by sgharms** [comment]
@thuyanduong-flatiron Merged! Thanks

**Pull request merged**